cmd/sriovdp/manager.go

```diff
@@ -100,6 +106,77 @@ func (rm *resourceManager) readConfig() error {
 	return nil
 }
 
+func (rm *resourceManager) storeConfig() error {
+	filename := filepath.Base(rm.configFile)
+	source, err := os.Open(rm.configFile)
+	if err != nil {
+		glog.Errorf("storeConfig(): error creating reading config file: %v", err)
+		return err
+	}
+	defer source.Close()
+
+	destination, err := os.Create(filepath.Join(hostConfigDir, filename))
+	if err != nil {
+		glog.Errorf("storeConfig(): error creating creating config file on host: %v", err)
+		return err
+	}
+	defer destination.Close()
+
+	_, err = io.Copy(destination, source)
+	if err != nil {
+		glog.Errorf("storeConfig(): error during copy config file to host: %v", err)
+		return err
+	}
+	return nil
+}
+
+func (rm *resourceManager) createCdiSpec() error {
+	glog.Infof("number of config: %d\n", len(rm.configList))
+	deviceAllocated := make(map[string]bool)
+	for _, rc := range rm.configList {
```

Comment:
This was copied for L184, why not just put into helper function to reduce code dupl ?